component/webui/src/main/java/org/exoplatform/social/webui/space/access/SpaceAccessApplicationLifecycle.java

```diff
@@ -132,7 +135,15 @@ private void addMembershipToIdentity(String remoteId, Space space) {
 
   
   private void processSpaceAccess(PortalRequestContext pcontext, String remoteId, Space space) throws IOException {
-    //
+
+    org.exoplatform.social.core.identity.model.Identity viewerIdentity = Utils.getViewerIdentity();
```

Comment:
after merging [https://github.com/Meeds-io/social/pull/457/files](https://github.com/Meeds-io/social/pull/457) Util.isExternal to be called instead